code/modules/jobs/job_types/roguetown/refugees/types/new/rogue.dm

```diff
@@ -0,0 +1,114 @@
+	..()
+	H.adjust_blindness(-3)
+	var/classes = list("Thief", "Smuggler", "Cutthroat")
+	var/classchoice = input("Choose your archetypes", "Available archetypes") as anything in classes
+
+	switch(classchoice)
+
+		if("Thief")
+			H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/swimming, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/climbing, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/craft/crafting, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/stealing, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/lockpicking, 4, TRUE)
+			shoes = /obj/item/clothing/shoes/roguetown/armor
+			neck = /obj/item/storage/belt/rogue/pouch/coins/poor
+			pants = /obj/item/clothing/under/roguetown/trou/leather
+			shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt
+			gloves = /obj/item/clothing/gloves/roguetown/fingerless
+			belt = /obj/item/storage/belt/rogue/leather
+			cloak = /obj/item/clothing/cloak/raincloak/mortus
+			backl = /obj/item/storage/backpack/rogue/satchel
+			beltr = /obj/item/rogueweapon/huntingknife/idagger
+			backpack_contents = list(/obj/item/lockpickring/mundane = 1)
+			H.change_stat("strength", -1)
+			H.change_stat("perception", 1)
+			H.change_stat("speed", 3)
+			H.change_stat("intelligence", 1)
+			ADD_TRAIT(H, TRAIT_DODGEEXPERT, TRAIT_GENERIC)
+			ADD_TRAIT(H, TRAIT_SEEPRICES_SHITTY, TRAIT_GENERIC)
+
+		if("Smuggler")
+			H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE) 
+			H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/swimming, 4, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/climbing, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 2, TRUE) 
+			H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
+			head =/obj/item/clothing/head/roguetown/helmet/bandana
+			shoes = /obj/item/clothing/shoes/roguetown/armor/leather
+			neck = /obj/item/storage/belt/rogue/pouch/coins/mid
+			pants = /obj/item/clothing/under/roguetown/trou/leather
+			shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt
+			gloves = /obj/item/clothing/gloves/roguetown/fingerless
+			belt = /obj/item/storage/belt/rogue/leather
+			armor = /obj/item/clothing/suit/roguetown/armor/leather
+			backl = /obj/item/storage/backpack/rogue/backpack
+			backr = /obj/item/gun/ballistic/revolver/grenadelauncher/crossbow
+			beltl = /obj/item/ammo_holder/quiver
```

Comment:
might want to change it to /obj/item/ammo_holder/quiver/bolts if you want them to start with ammo